playground/infrastructure/ci_helper.py

```diff
@@ -0,0 +1,132 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import os
+from typing import List
+
+from api.v1.api_pb2 import Sdk, SDK_JAVA, SDK_GO, SDK_PYTHON, SDK_SCIO
+
+possible_sdk = {'java': SDK_JAVA, 'go': SDK_GO, 'py': SDK_PYTHON, 'scala': SDK_SCIO}
+
+
+def _find_examples(work_dir: str) -> List[str]:
```

Comment:
I see that all these functions outside of the `CIHelper` class we use only here, if I'm not mistaken, the better pattern is to move these functions under the `CIHelper` class.

Reply:
Changed.